src/test/scala/org/ergoplatform/modifiers/mempool/ErgoTransactionSpec.scala

```diff
@@ -543,4 +543,46 @@ class ErgoTransactionSpec extends ErgoPropertyTest with ErgoTestConstants {
     }
   }
 
+  property("context extension with neg id") {
+    val negId: Byte = -1
+
+    ADKey @@ Base16.decode("c95c2ccf55e03cac6659f71ca4df832d28e2375569cec178dcb17f3e2e5f7742").get
```

Comment:
This probably should be an explicit assertion.

Reply:
removed